{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T16:47:11.481Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:11:19.802Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:12:13.397Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:12:39.873Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:15:47.693Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:15:52.862Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:15:55.844Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:15:58.978Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:16:02.802Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:17:53.733Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:17:56.155Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:23:53.869Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:24:58.331Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-22T17:25:39.560Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-03-26T22:33:52.382Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T18:24:31.121Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:04:24.902Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:06:54.524Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:07:57.593Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:11:35.501Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:16:48.139Z"}
{"level":"error","message":"Unhandled Rejection at: TypeError: Cannot read properties of undefined (reading 'toLowerCase')\n    at C:\\Users\\gabia\\PFE---1\\routes\\connectionRoutes.js:185:58\n    at Layer.handle [as handle_request] (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at Function.handle (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:175:3)\n    at router (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:47:12)","timestamp":"2024-04-02T19:20:37.027Z"}
{"code":"ESOCKET","command":"CONN","errno":-4077,"level":"error","message":"Error sending email: read ECONNRESET","stack":"Error: read ECONNRESET\n    at TLSWrap.onStreamRead (node:internal/stream_base_commons:217:20)","syscall":"read","timestamp":"2024-04-02T19:21:25.987Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:41:27.562Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T19:41:37.087Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:22:23.686Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:22:27.389Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:23:27.949Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:23:31.764Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:23:44.623Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:29:30.694Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:31:04.517Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:31:10.396Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:31:38.688Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:31:47.390Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:31:54.734Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:32:03.446Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:32:23.371Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:32:41.530Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:33:07.826Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:33:29.280Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:33:31.587Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:34:08.949Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:34:30.312Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:34:34.124Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:34:39.627Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:35:03.585Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:35:51.416Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:35:57.997Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:38:11.354Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:38:25.078Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:38:46.920Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:39:06.010Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:39:12.702Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:39:31.960Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:39:35.498Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:39:37.848Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:39:43.403Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:40:06.888Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:40:11.865Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:40:48.661Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T20:40:55.490Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:18:06.191Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:18:22.651Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:19:29.668Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:22:38.966Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:22:43.702Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:22:57.181Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:23:00.772Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:23:07.071Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:25:43.686Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:25:47.980Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:26:04.453Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:26:32.224Z"}
{"level":"error","message":"ReferenceError: err is not defined\n    at isAdmin (C:\\Users\\gabia\\PFE---1\\middlewares\\roles.js:8:45)\n    at Layer.handle [as handle_request] (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:328:13)\n    at C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:286:9\n    at Function.process_params (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at authenticate (C:\\Users\\gabia\\PFE---1\\middlewares\\auth.js:17:5)\n    at Layer.handle [as handle_request] (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:328:13)\n    at C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\index.js:286:9","timestamp":"2024-04-02T21:26:49.571Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:27:07.420Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:27:20.414Z"}
{"level":"error","message":"ReferenceError: C:\\Users\\gabia\\PFE---1\\views\\index.ejs:27\n    25|                      <select id=\"tableName\" class=\"form-control\" style=\"width: 150px;\">\r\n    26|                      <option value=\"\">select table</option> <!-- Empty option added -->\r\n >> 27|                      <% if (tables && tables.length > 0) { %>\r\n    28|                        <% tables.forEach(table => { %>\r\n    29|                                 <!-- Make sure to reference the correct property -->\r\n    30|                           <option value=\"<%= table.Tables_in_fss %>\"><%= table.Tables_in_fss %></option>\r\n\ntables is not defined\n    at eval (\"C:\\\\Users\\\\gabia\\\\PFE---1\\\\views\\\\index.ejs\":21:8)\n    at index (C:\\Users\\gabia\\PFE---1\\node_modules\\ejs\\lib\\ejs.js:703:17)\n    at tryHandleCache (C:\\Users\\gabia\\PFE---1\\node_modules\\ejs\\lib\\ejs.js:274:36)\n    at exports.renderFile [as engine] (C:\\Users\\gabia\\PFE---1\\node_modules\\ejs\\lib\\ejs.js:491:10)\n    at View.render (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\view.js:135:8)\n    at tryRender (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\application.js:657:10)\n    at Function.render (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\application.js:609:3)\n    at ServerResponse.render (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\response.js:1039:7)\n    at isAdmin (C:\\Users\\gabia\\PFE---1\\middlewares\\roles.js:9:16)\n    at Layer.handle [as handle_request] (C:\\Users\\gabia\\PFE---1\\node_modules\\express\\lib\\router\\layer.js:95:5)","timestamp":"2024-04-02T21:27:21.846Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:27:40.751Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-02T21:27:59.779Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-15T17:38:52.888Z"}
{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"fatal":true,"level":"error","message":"Error connecting to MySQL: ER_ACCESS_DENIED_ERROR: Access denied for user 'admin'@'localhost' (using password: NO)","sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000","stack":"Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'admin'@'localhost' (using password: NO)\n    at Sequence._packetToError (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\sequences\\Sequence.js:47:14)\n    at Handshake.ErrorPacket (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\sequences\\Handshake.js:123:18)\n    at Protocol._parsePacket (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:291:23)\n    at Parser._parsePacket (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Parser.js:433:10)\n    at Parser.write (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Parser.js:43:10)\n    at Protocol.write (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:38:16)\n    at Socket.<anonymous> (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\Connection.js:88:28)\n    at Socket.<anonymous> (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\Connection.js:526:10)\n    at Socket.emit (node:events:518:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    --------------------\n    at Protocol._enqueue (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:144:48)\n    at Protocol.handshake (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:51:23)\n    at Connection.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\Connection.js:116:18)\n    at Object.<anonymous> (C:\\Users\\gabia\\PFE---2\\model\\dbConfig.js:34:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Module.require (node:internal/modules/cjs/loader:1235:19)\n    at require (node:internal/modules/helpers:176:18)","timestamp":"2024-04-15T17:38:52.989Z"}
{"level":"error","message":"Unable to connect to the database: Access denied for user 'admin'@'localhost' (using password: NO)","name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'admin'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:94:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32","timestamp":"2024-04-15T17:38:52.998Z"}
{"level":"error","message":"Error syncing models: Access denied for user 'admin'@'localhost' (using password: NO)","name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'admin'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:94:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32","timestamp":"2024-04-15T17:38:53.000Z"}
{"level":"error","message":"Error syncing models: Access denied for user 'admin'@'localhost' (using password: NO)","name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'admin'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'admin'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:94:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32","timestamp":"2024-04-15T17:38:53.006Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-15T17:39:34.024Z"}
{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"fatal":true,"level":"error","message":"Error connecting to MySQL: ER_ACCESS_DENIED_ERROR: Access denied for user 'user'@'localhost' (using password: NO)","sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000","stack":"Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'user'@'localhost' (using password: NO)\n    at Sequence._packetToError (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\sequences\\Sequence.js:47:14)\n    at Handshake.ErrorPacket (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\sequences\\Handshake.js:123:18)\n    at Protocol._parsePacket (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:291:23)\n    at Parser._parsePacket (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Parser.js:433:10)\n    at Parser.write (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Parser.js:43:10)\n    at Protocol.write (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:38:16)\n    at Socket.<anonymous> (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\Connection.js:88:28)\n    at Socket.<anonymous> (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\Connection.js:526:10)\n    at Socket.emit (node:events:518:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    --------------------\n    at Protocol._enqueue (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:144:48)\n    at Protocol.handshake (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\protocol\\Protocol.js:51:23)\n    at Connection.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\mysql\\lib\\Connection.js:116:18)\n    at Object.<anonymous> (C:\\Users\\gabia\\PFE---2\\model\\dbConfig.js:34:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Module.require (node:internal/modules/cjs/loader:1235:19)\n    at require (node:internal/modules/helpers:176:18)","timestamp":"2024-04-15T17:39:34.076Z"}
{"level":"error","message":"Unable to connect to the database: Access denied for user 'user'@'localhost' (using password: NO)","name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'user'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:94:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32","timestamp":"2024-04-15T17:39:34.088Z"}
{"level":"error","message":"Error syncing models: Access denied for user 'user'@'localhost' (using password: NO)","name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'user'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:94:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32","timestamp":"2024-04-15T17:39:34.093Z"}
{"level":"error","message":"Error syncing models: Access denied for user 'user'@'localhost' (using password: NO)","name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'user'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'user'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\mysql\\connection-manager.js:94:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:222:24)\n    at async C:\\Users\\gabia\\PFE---2\\node_modules\\sequelize\\lib\\dialects\\abstract\\connection-manager.js:174:32","timestamp":"2024-04-15T17:39:34.095Z"}
{"level":"error","message":"This error will be logged to the console, error.log, and combined.log","timestamp":"2024-04-15T17:41:54.812Z"}
{"level":"error","message":"ReferenceError: C:\\Users\\gabia\\PFE---2\\connection\\messages.ejs:17\n    15| <div class=\"container\">\r\n    16|   <div class=\"custom-jumbotron\">\r\n >> 17|       <h1 class=\"display-4\">Hello, <%= NAME %></h1>\r\n    18|       <p class=\"lead\">Registration successful! Please check your email (<strong style=\"color: green;\"><%= EMAIL %></strong>) for the confirmation link.</p>\r\n    19|       <hr class=\"my-4\">\r\n    20|       <p>After confirming your email, you can click the button below to log in.</p>\r\n\nNAME is not defined\n    at eval (\"C:\\\\Users\\\\gabia\\\\PFE---2\\\\connection\\\\messages.ejs\":12:26)\n    at messages (C:\\Users\\gabia\\PFE---2\\node_modules\\ejs\\lib\\ejs.js:703:17)\n    at tryHandleCache (C:\\Users\\gabia\\PFE---2\\node_modules\\ejs\\lib\\ejs.js:274:36)\n    at exports.renderFile [as engine] (C:\\Users\\gabia\\PFE---2\\node_modules\\ejs\\lib\\ejs.js:491:10)\n    at View.render (C:\\Users\\gabia\\PFE---2\\node_modules\\express\\lib\\view.js:135:8)\n    at tryRender (C:\\Users\\gabia\\PFE---2\\node_modules\\express\\lib\\application.js:657:10)\n    at Function.render (C:\\Users\\gabia\\PFE---2\\node_modules\\express\\lib\\application.js:609:3)\n    at ServerResponse.render (C:\\Users\\gabia\\PFE---2\\node_modules\\express\\lib\\response.js:1048:7)\n    at C:\\Users\\gabia\\PFE---2\\routes\\connectionRoutes.js:110:18\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-04-15T17:42:43.424Z"}
